test/jdk/java/awt/PopupMenu/PopupMenuLocation.java

```diff
@@ -33,21 +33,25 @@
 import java.awt.Robot;
 import java.awt.Toolkit;
 import java.awt.event.InputEvent;
+import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
-import java.awt.event.*;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+import javax.swing.SwingUtilities;
 
 /**
  * @test
  * @key headful
  * @bug 8160270
- * @run main/timeout=300 PopupMenuLocation
+ * @run main/timeout=300/othervm -Dsun.java2d.uiScale=1 PopupMenuLocation
```

Comment:
Why do you need a "-Dsun.java2d.uiScale=1" option? I do not think it affects this test execution in any way.

Reply:
Removed it